fix: suppress spammy AVC denials from Trivalent reading /proc

HastD opened the pull request:
The Trivalent launch script reads `/proc` to determine whether an instance of Trivalent is already running, and this results in a bunch of irrelevant SELinux denials from attempting to read other processes' info. This `dontaudit` rule suppresses those messages.

RoyalOughtness commented:
@HastD 

```

[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => Compiling targeted trivalent module
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => trivalent.te:318:ERROR 'unrecognized character' at token ''' on line 20046:
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => '
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => #line 318
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => /usr/bin/checkmodule:  error(s) encountered while parsing configuration
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => make: *** [/usr/share/selinux/devel/include/Makefile:157: tmp/trivalent.mod] Error 1
```

HastD committed:
fix: suppress spammy AVC denials from Trivalent reading /proc

The Trivalent launch script reads `/proc` to determine whether an
instance of Trivalent is already running, and this results in a bunch of
irrelevant SELinux denials from attempting to read other processes'
info. This `dontaudit` rule suppresses those messages.

Signed-off-by: Daniel Hast <hast.daniel@protonmail.com>

HastD commented:
@RoyalOughtness Should be fixed now. Apparently .te files are parsed in such a way that `'` characters in comments can cause issues in some contexts...

RoyalOughtness committed:
Merge branch 'live' into trivalent-dontaudit
